### PR TITLE
chore: i18n

### PR DESCRIPTION
**Pull request by nuanyang233**
chore: i18n

## Description

Closes #MF-1740

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/plugins/Web3Profile/src/locales/en-US.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "Flow_wallet": "Flow Wallet",
     "wallet_status_button_change": "Change",
     "plugin_wallet_on_connect": "Connect Wallet",
-    "wallet_verify_has_bound": "The wallet has been bound. Please switch wallets.",
+    "wallet_verify_has_bound": "The wallet has already been bound to {currentPersona} Please switch to another wallet.",
```
Comment (UncleBill) [inline packages/plugins/Web3Profile/src/locales/en-US.json]:
```suggestion
    "wallet_verify_has_bound": "The wallet has already been bound to {currentPersona} Please switch to another.",
```

File: packages/mask/shared-ui/locales/en-US.json [hunk 2]
```diff
@@ -988,7 +988,7 @@
     "plugin_game_name": "GAME",
     "plugin_game_description": "Play game with your NFT.",
     "wallet_verify_persona_sign": "Persona Sign",
-    "wallet_verify_has_bound": "The wallet has been bound. Please switch wallets.",
+    "wallet_verify_has_bound": "The wallet has already been bound to {currentPersona} Please switch to another wallet.",
```
Comment (UncleBill) [inline packages/mask/shared-ui/locales/en-US.json]:
```suggestion
    "wallet_verify_has_bound": "The wallet has already been bound to {currentPersona} Please switch to another.",
```

**Review by UncleBill — CHANGES_REQUESTED** [review]

File: packages/plugins/Web3Profile/src/locales/en-US.json (hunk 1 above)
Comment (nuanyang233) [reply packages/plugins/Web3Profile/src/locales/en-US.json]: This is the confirmed text, no need to change.

File: packages/mask/shared-ui/locales/en-US.json (hunk 2 above)
Comment (guanbinrui) [inline packages/mask/shared-ui/locales/en-US.json]:
```suggestion
    "wallet_verify_has_bound": "The wallet has already been bound to {currentPersona}. Please switch to another wallet.",
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**